build.sbt

```diff
@@ -1,33 +1,32 @@
 import Dependencies._
 
-name := "Tezos-FullStack-Console"
+lazy val root = (project in file(".")).settings(
+  name := "Tezos-FullStack-Console",
+  version := "0.1",
+  scalaVersion := "2.12.8",
+  resolvers += "scalaz-bintray" at "https://dl.bintray.com/scalaz/releases",
+  resolvers += "typesafe" at "http://repo.typesafe.com/typesafe/releases/",
+  resolvers += "Scalac" at "https://raw.githubusercontent.com/ScalaConsultants/mvn-repo/master/",
+
+  libraryDependencies ++= akkaHttp ++ akkaHttpCors ++ akkaStream ++ cats ++ courier ++ greenMail ++ jodaTime ++ logBack ++
+    mySql ++ pureConfig ++ scalactic ++ scalaTest ++ sl4j ++ slick ++ tesozFCTM ++ wireMock ++ testContainers,
+  
+    Defaults.itSettings,
+
+    parallelExecution in IntegrationTest := true,
+    // No need to run tests while building jar
+    test in assembly := {},
```

Comment:
rly not? even for relelase

Reply:
i just copied this part. What would you recommend here ?

Reply:
just asking :) it looks a bit strange, like somebody had problems with tests and wanted to build jars quickly

Reply:
I would say this practice is quite common

Reply:
Checked `defaultAssemblySettings` from Assembly and indeed it has overwritten `test`